pages/community/index.js

```diff
@@ -110,225 +47,149 @@ function CommunityIndexPage() {
-          </a>
-        </div>
-        <div className="h-full cursor-pointer rounded-lg shadow-lg bg-usecases bg-cover bg-center mt-10 relative md:w-[49%] md:mt-0">
-          <a
-            href="https://github.com/orgs/asyncapi/discussions/593"
-            target="_blank"
-            rel="noreferrer"
-          >
-            <div>
-              <div className="absolute p-2 bg-primary-600 text-white font-bold rounded-tl-lg">
-                Coming Soon
-              </div>
-              <div className="w-full h-full absolute rounded-lg" />
-              <div className="w-full h-full flex flex-col justify-end absolute">
-                <div className="bg-yellow-600 text-left rounded-b-lg p-5 ">
-                  <Heading
-                    level="h2"
-                    typeStyle="heading-md"
-                    className="text-white"
-                  >
-                    Recognize AsyncAPI OSS contributors
-                  </Heading>
-                  <Heading
-                    level="h2"
-                    typeStyle="body-md"
-                    textColor="text-white"
-                    className="text-sm"
-                  >
-                    Recognize the people who inspire and contribute to our OSS
-                    community.
-                  </Heading>
-                </div>
-              </div>
-            </div>
-          </a>
+                  <a href={event.url} className="flex">
+                    <div className="flex justify-center flex-col w-24 md:w-40 bg-pink-400 p-1 md:p-4 rounded-md text-xs md:text-sm font-semibold text-white">
+                      <span className="text-center">
+                        {event.title.split(' ')[0]}
+                      </span>
+                    </div>
+                    <div className="ml-4 text-left">
+                      <h1 className="text-md md:text-lg">{event.title}</h1>
+                      <span className="text-gray-500 text-xs md:text-sm">
+                        {event.date.local().format('LLLL')} UTC
+                        {event.date.local().format('Z')}
+                      </span>
+                    </div>
+                  </a>
+                </li>
+              );
+            })}
+          </ul>
         </div>
       </div>
-      <div className="z-40 mt-[30rem] md:mt-10 bg-white w-full md:h-130 rounded-lg shadow-xl md:flex md:justify-between">
-        <div className="p-10 md:flex justify-between md:w-2/5 md:h-full flex-col text-center md:text-left">
+      <div className="md:flex md:justify-between w-full mt-20">
+        <div className="md:w-[45%] flex flex-col justify-between mt-10 md:mt-0">
           <div>
-            <Heading
-              level="h2"
-              typeStyle="heading-md"
-              textColor="text-purple-300"
-            >
-              All community info, tracked
-            </Heading>
+            <Card
+              type="small"
+              tagline="Community"
```

Comment:
```suggestion
              tagline="Goals"
```

🤔  Isn't goals a more appropriate tagline since this card is about the community **goals**?